cypress-tests/cypress/e2e/PaymentTest/00019-UPI.cy.js

```diff
@@ -0,0 +1,177 @@
+import confirmBody from "../../fixtures/confirm-body.json";
+import createPaymentBody from "../../fixtures/create-payment-body.json";
+import refundBody from "../../fixtures/refund-flow-body.json";
+import State from "../../utils/State";
+import getConnectorDetails, * as utils from "../PaymentUtils/utils";
+
+let globalState;
+
+describe("UPI Payments - Hyperswitch", () => {
+  let should_continue = true; // variable that will be used to skip tests if a previous test fails
+
+  context("[Payment] [UPI - UPI Collect] Create & Confirm + Refund", () => {
+    before("seed global state", () => {
```

Comment:
We need hooks to be added at context level just because the State needs to be updated prior to upcoming context. Adding that at describe level will not run before each context. We cannot use `beforeEach` here either as it will run before every `it()` resulting in tests to fail.